lib/process-services-cloud/src/lib/group/components/group-cloud.component.ts

```diff
@@ -121,9 +126,10 @@
     invalidGroups: IdentityGroupModel[] = [];
 
     searchGroups$ = new BehaviorSubject<IdentityGroupModel[]>(this.searchGroups);
-    _subscriptAnimationState = 'enter';
+    _subscriptAnimationState: string = 'enter';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Alfresco/alfresco-ng2-components/pullRequest?prid=8980729)
```suggestion
    _subscriptAnimationState: string = "enter";
```